### PR TITLE
Simple code change to eliminate deprecated warning (See issue #99)

### PR DESCRIPTION
**Pull request by driverjb**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
    - `npm run benchmark` does not exist
- [x] tests and/or benchmarks are included
```text
λ npm run test

> @fastify/accepts-serializer@5.2.0 test
> npm run test:unit && npm run test:typescript


> @fastify/accepts-serializer@5.2.0 test:unit
> tap

​ PASS ​ test/test.js 55 OK 227.309ms
​


  🌈 SUMMARY RESULTS 🌈

​
Suites:   ​1 passed​, ​1 of 1 completed​
Asserts:  ​​​55 passed​, ​of 55​
​Time:​   ​6s​
----------------------|---------|----------|---------|---------|-------------------
File                  | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
----------------------|---------|----------|---------|---------|-------------------
All files             |     100 |      100 |     100 |     100 |
 SerializerManager.js |     100 |      100 |     100 |     100 |
 index.js             |     100 |      100 |     100 |     100 |
----------------------|---------|----------|---------|---------|-------------------

> @fastify/accepts-serializer@5.2.0 test:typescript
> tsd
```
- [x] documentation is changed or added
    - Not applicable in this change
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fastify/accepts-serializer",
-  "version": "5.2.0",
+  "version": "5.3.0",
```
Comment (climba03003) [inline package.json:3]: Please do not change the version here.
It should be handled by the releaser.

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (driverjb) [reply package.json:3]: Do you want me to submit a new PR or can we change it here directly?

Comment (driverjb) [inline package.json:3]:
```suggestion
  "version": "5.2.0",
```

**Review by driverjb — COMMENTED** [review]
I've modified the version back to the original value of 5.2.0

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**